Added the unique files in the prev repo

BikramKumar6928 opened the pull request.

BikramKumar6928 committed:
Added the unique files in the prev repo

premsagar279 reviewed (changes requested):
Please remove ajax,php from controller and then  merge

BikramKumar6928 committed:
commented out ajax.php

BikramKumar6928 committed:
added index.php to config part

changed the corrresponding parts for correct loading in header and home

Added bulkorder

premsagar279 committed:
Bug fixed

19901990z commented:
Hello my friend .. I sent you an email on your email
